This file is part of MXE.
See index.html for further information.

This patch ensures that the correct dependencies can always be found for
static mingw builds. Submitted upstream: 

   https://issues.apache.org/jira/browse/LOGCXX-381

diff -NBaur vanilla/liblog4cxx.pc.in mingw_cross/liblog4cxx.pc.in
--- vanilla/liblog4cxx.pc.in	2008-04-01 07:34:52.000000000 +0900
+++ mingw_cross/liblog4cxx.pc.in	2011-05-09 10:31:00.175584637 +0900
@@ -20,6 +20,7 @@
 Name: log4cxx
 Description: log4cxx C++ logging framework
 Version: @VERSION@
+Requires: apr-util-1, apr-1
 Libs: -L${libdir} -llog4cxx
 Cflags: -I${includedir}
 
